web/src/beta/lib/core/mantle/atoms/compute.ts

```diff
@@ -208,21 +215,19 @@ export function computeAtom(cache?: typeof globalDataFeaturesCache) {
         layerId: currentLayer.id,
       });
 
-      const computedLayer = await evalLayer(currentLayer, {
-        getAllFeatures: async () => undefined,
-        getFeatures: async () => undefined,
-      });
-
-      if (!computedLayer) {
-        return;
-      }
+      const computedLayer = value.needComputingLayer
+        ? await evalLayer(currentLayer, {
+            getAllFeatures: async () => undefined,
+            getFeatures: async () => undefined,
+          })
+        : undefined;
```

Comment:
In almost use case of writeComputedFeaturee, evaluating the layer is unnecessary. So I added this option.